docs/source/en/model_doc/beit.mdx

```diff
@@ -59,6 +59,12 @@ Tips:
   `use_relative_position_bias` attribute of [`BeitConfig`] to `True` in order to add
   position embeddings.
 
+<Tip warning={true}>
+
+BEiT does masked image modeling by predicting visual tokens of a Vector-Quantize Variational Autoencoder (VQ-VAE), whereas other vision models like ViT and DeiT predict RGB pixel values. The [`AutoModelForMaskedImageModeling`] class supports pixel-based image modeling, so you will need to use [`BeitForMaskedImageModeling`] directly if you wish to do masked image modeling with BEiT.
```

Comment:
Not sure if this should be on the main doc page or within the docstring for the `BeitForMaskedImageModeling` class. Happy to move it if you want!